test/unit/io/structure_file/format_vienna_test.cpp

```diff
@@ -107,20 +108,29 @@ struct read : public ::testing::Test
     {
         std::stringstream istream{input};
 
-        auto field_set = fields<field::id, field::seq, field::bpp, field::structure, field::energy>{};
-        structure_file_input fin{istream, format_vienna{}, field_set};
+        auto field_set = seqan3::fields<seqan3::field::id,
+                                        seqan3::field::seq,
+                                        seqan3::field::bpp,
+                                        seqan3::field::structure,
+                                        seqan3::field::energy>{};
+        seqan3::structure_file_input fin{istream, seqan3::format_vienna{}, field_set};
         fin.options = options;
 
         auto it = fin.begin();
         for (size_t idx = 0ul; idx < expected_seq.size(); ++idx, ++it)
         {
-            EXPECT_EQ(check_energy, get<field::energy>(*it).has_value());
-            if (check_seq)       { EXPECT_TRUE((std::ranges::equal(get<field::seq>(*it), expected_seq[idx]))); }
-            if (check_id)        { EXPECT_TRUE((std::ranges::equal(get<field::id>(*it), expected_id[idx]))); }
-            if (check_structure) { bpp_test(get<field::bpp>(*it), expected_interactions[idx]); }
-            if (check_energy)    { EXPECT_DOUBLE_EQ(*get<field::energy>(*it), expected_energy[idx]); }
+            EXPECT_EQ(check_energy, seqan3::get<seqan3::field::energy>(*it).has_value());
+            if (check_seq)
+                {EXPECT_TRUE((std::ranges::equal(seqan3::get<seqan3::field::seq>(*it), expected_seq[idx])));}
```

Comment:
sorry one more thing. For **single line** if clauses you can omit the braces
```suggestion
                EXPECT_TRUE((std::ranges::equal(seqan3::get<seqan3::field::seq>(*it), expected_seq[idx])));
```

Reply:
If I do that, I get the following error messages:

```bash
...seqan3/test/unit/io/structure_file/format_vienna_test.cpp: In member function 'void read::do_read_test(const string&)':
...seqan3/test/unit/io/structure_file/format_vienna_test.cpp:123:16: error: suggest explicit braces to avoid ambiguous 'else' [-Werror=dangling-else]
  123 |             if (check_seq)
      |                ^
...seqan3/test/unit/io/structure_file/format_vienna_test.cpp:125:16: error: suggest explicit braces to avoid ambiguous 'else' [-Werror=dangling-else]
  125 |             if (check_id)
      |                ^
...seqan3/test/unit/io/structure_file/format_vienna_test.cpp:129:16: error: suggest explicit braces to avoid ambiguous 'else' [-Werror=dangling-else]
  129 |             if (check_energy)
      |                ^
...seqan3/test/unit/io/structure_file/format_vienna_test.cpp:131:16: error: suggest explicit braces to avoid ambiguous 'else' [-Werror=dangling-else]
  131 |             if (check_structure)
```

Reply:
The macro expands to multiple lines, so doing braces is correct

Reply:
ah ok :+1: